CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fix the issue where config default setting command normalized existing vim's keybindings (e.g. `<space>` to ` `) and can cause duplicate keybindings.
+- `<spc b Y>` doesn't change cursor position
```

Comment:
nit: `<spc> b Y`
Maybe it's clearer to say "Keep cursor position after `<spc> b Y`"?

Reply:
Yes, I will update this